www/artists/mbolen/gallery.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Where is the closing tag for this? It seems that you open the html tag, but then leave it open. I don't know why CircleCI isn't flaging that, but it seems like a rather big problem.